app/assets/stylesheets/application/member-card.scss

```diff
@@ -1,16 +1,32 @@
+.row {
```

Comment:
here are 2 dangerous things:
- you are applying css to a bootstrap class
- the most important thing is that this class is widely used throughout the entire app, so it is very likely that it could have an undesired impact somewhere.